release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.Text.Encoding.CodePages.md

```diff
@@ -0,0 +1,14 @@
+# System.Text.Encoding.CodePages
+
```

Comment:
@Anipik 
This is not including the APIs we added during 5.0. Is this expected?

Reply:
this should include all the new apis that we added in this package during the 5.0 timeframe, if u give me a missing api, i can take a look